lib/policies/load-balancing.js

```diff
@@ -209,20 +209,22 @@ DCAwareRoundRobinPolicy.prototype.newQueryPlan = function (keyspace, queryOption
     this.index = 0;
   }
   this._sliceNodesByDc();
+  // Use a local reference of hosts
+  var localHostsArray = this.localHostsArray;
```

Comment:
You've got it, sorry for not being more descriptive in the ticket... I've should have included more info other than title and stack trace.

I've just copied your description into the ticket for future reference.